bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -209,4 +231,10 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="powerState">
+		<item-type>String</item-type>
+		<label>Power State</label>
+		<description>Power State of the TV</description>
+		<state readOnly="true"/>
```

Comment:
If you know at least some of the available states, you could add them as options here as this would both document them and make them available in the UI (and translateable):
```suggestion
		<state readOnly="true">
		    <options>
		        <option value="PowerOn">Power On</option>
		        <option value="DisplayOff">Display Off</option>
		    </options>
		</state>
```

etc.